src/univers/nuget.py

```diff
@@ -0,0 +1,163 @@
+#
+# Copyright (c) nexB Inc. and others.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Visit https://aboutcode.org and https://github.com/nexB/univers for support and download.
+
+import functools
+import re
+
+import semver
```

Comment:
Could we eventually avoid having two semver libraries and reuse `semantic-version` we already use here?
 @rbarrois https://pypi.org/project/semantic-version/ https://github.com/rbarrois/python-semanticversion that we use can parse node-semver version specs constraints while https://github.com/python-semver/python-semver/ can only deal with versions and not the npm specs (we use to convert from npm native to our vers syntax)  which is why we picked in the first place.

Reply:
I tried reusing the existing library at first place, It returns prerelease and build as tuples, whereas the newly imported one returns a string, and there is no concrete way to get to strings from those tuples.
For example:
```
semantic_version.Version.coerce("1.0.0.0").prerelease
('0',)
nuget.Version.from_string("1.0.0.0").prerelease
None
```
That's why I think we should use the new library for nuget

Reply:
One more thing I observed here, the equal comparison of both the libraries are different, I tried to run all the tests after making the required adjustments by using the `semantic_version` library, but some corner cases are failing there, one more reason to use the new library for nuget

Reply:
Here is the gist for same https://gist.github.com/TG1999/724c868fa21882aa880c091cf5e1964d

Reply:
"1.0.0", "1.0.0+beta"

Reply:
> "1.0.0", "1.0.0+beta"

So if I get you, nuget consider that these tow are equal? https://github.com/nexB/univers/pull/60/files#diff-1b35636a23b0127a5ed3b726fe14efdf9df1a159e27dd87ead2be71547b78cccR39

This is mind blowing!

*sigh* ok, well, so let's use this for now. :|